pylint/pyreverse/main.py

```diff
@@ -192,6 +192,22 @@
             "help": "Use colored output. Classes/modules of the same package get the same color.",
         },
     ),
+    (
+        "max-depth",
+        {
+            "dest": "max_depth",
+            "action": "store",
+            "default": None,
+            "metavar": "<depth>",
+            "type": "int",
+            "group": OPTIONS_GROUPS["DISPLAY"],
```

Comment:
```suggestion
            "group": OPTIONS_GROUPS["FILTERING"],
```
I think the new option fits the FILTERING group better, as it is similar to e.g. `--show-associated`. If you agree, can you move it up near the other filtering options, as the options are also ordered in the source code accordingly?